com.obsproject.Studio.Plugin.RewardsTheater.metainfo.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="addon">
+  <id>com.obsproject.Studio.Plugin.RewardsTheater</id>
+  <extends>com.obsproject.Studio</extends>
+  <name>RewardsTheater</name>
+  <developer_name>gottagofaster</developer_name>
+  <summary>An OBS plugin that lets your viewers redeem videos or sounds on stream via Twitch Channel Points</summary>
+  <url type="homepage">https://github.com/gottagofaster236/RewardsTheater</url>
+  <url type="help">https://github.com/gottagofaster236/RewardsTheater/blob/master/README.md</url>
+  <url type="bugtracker">https://github.com/gottagofaster236/RewardsTheater/issues</url>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0</project_license>
```

Comment:
this license is not consistent with the headers
https://github.com/gottagofaster236/RewardsTheater/commit/b54ae95b6ae8c4f6897ae7c27053df3406f57591

Reply:
nevermind it's just the old SPDX.